recipes/lammps/a100/environments.yaml

```diff
@@ -0,0 +1,36 @@
+lammps:
+  compiler:
+      - toolchain: gcc
+        spec: gcc@11
+  mpi:
+      spec: cray-mpich
+      gpu: cuda
+  unify: when_possible
+  specs:
+  - cmake
+  - cuda@11.8
+  - hwloc
+  - python@3.10
+  - vtk
+  - kokkos
+  - blas
+  - lapack
+  - fftw-api
+  - lammps@develop +cuda
+  - lammps@develop +plumed +cuda
+  variants:
+  - +mpi
+  - +cuda
+  - cuda_arch=80
+  views:
+    develop:
+      exclude: ['lammps']
+    lammps:
+      exclude: ['lammps+plumed']
+      link: run
+    plumed:
```

Comment:
You'll have to exclude the `lammps` that does not have `plumed` enabled, which would likely be `lammps~plumed`